src/data/html-quiz.ts

```diff
@@ -1,4 +1,34 @@
 const htmlQuiz = [
+  {
+    Question: "How many Heading tag are there in HTML5",
+    Answer: "6",
+    Distractor1: "4",
+    Distractor2: "8",
+    Distractor3: "6",
+    Explanation:
+      "There are 6 Heading tag in HTML5  by default",
+    Link: "https://youtu.be/kUMe1FH4CHE?t=1219",
+  },
+  {
+    Question: "What is the font-size of the h1 heading tag",
+    Answer: "2em",
+    Distractor1: "2em",
+    Distractor2: "3.4em",
+    Distractor3: "2.7em",
+    Explanation:
+      "Font-Size of h1 tag is 2em by default",
+    Link: "https://www.freecodecamp.org/news/how-to-change-text-size-in-html/",
+  },
+  {
+    Question: "How is document type initialized in HTML5",
+    Answer: "<!DOCTYPE HTML>",
+    Distractor1: "</DOCTYPE HTML>",
+    Distractor2: "</DOCTYPE>",
+    Distractor3: "<!DOCTYPE HTML>",
```

Comment:
distractors are supposed to be for the wrong answers

```suggestion
    Distractor3: "<!DOCTYPE HTML>>>",
```